client/my-sites/hosting/github/disconnect-github-expander/index.tsx

```diff
@@ -15,40 +13,20 @@ interface DisconnectGitHubExpanderProps {
 
 export function DisconnectGitHubExpander( { connection }: DisconnectGitHubExpanderProps ) {
 	const { __ } = useI18n();
-	const panelId = useInstanceId( DisconnectGitHubExpander, 'disconnect-github' ) + '-panel';
 	const dispatch = useDispatch();
-	const [ isExpanded, setIsExpanded ] = useState( false );
-
 	// Using ReactQuery to manage `isDisconnecting` state because it's not exposed from the Redux store.
 	const mutation = useMutation< unknown, unknown, Connection >(
 		async ( c ) => await dispatch( deleteStoredKeyringConnection( c ) )
 	);
 	const { mutate: disconnect, isLoading: isDisconnecting } = mutation;
 
 	return (
-		<>
-			<button
-				className="disconnect-github-expander__button"
-				aria-controls={ panelId }
-				aria-expanded={ isExpanded }
-				onClick={ () => setIsExpanded( ( expanded ) => ! expanded ) }
-			>
-				<Gridicon icon="chevron-right" size={ 16 } /> { __( 'Disconnect from GitHub' ) }
-			</button>
-			<div
-				id={ panelId }
-				className="disconnect-github-expander__panel"
-				style={ { display: isExpanded ? 'block' : 'none' } }
-			>
-				<Button
-					scary
-					onClick={ () => disconnect( connection ) }
-					busy={ isDisconnecting }
-					disabled={ isDisconnecting } // `busy` doesn't actually disable the button
-				>
-					{ __( 'Disconnect' ) }
-				</Button>
-			</div>
-		</>
+		<Button
+			onClick={ () => disconnect( connection ) }
+			busy={ isDisconnecting }
```

Comment:
I think we should always display this button as `busy` so it represents a destructive action.

Reply:
is that a common pattern? this seems like it would draw more attention to this button

Reply:
It is... most CTAs are red. We can either make it busy all the time or remove the border, making it less prominent.

Reply:
Oh, sorry about that. I mistakenly thought of `busy` as the destructive, red button. But `busy` actually means  `loading` 🤦 

It's supposed to be `scary` and preferentially borderless so it doesn't draw too much attention, but it's still there.